pkgs/by-name/fa/fastnlo-toolkit/package.nix

```diff
@@ -35,6 +35,11 @@ stdenv.mkDerivation rec {
     ./yoda2_support.patch
   ];
 
+  nativeBuildInputs = [
+    lhapdf # lhapdf-config
+    yoda # yoda-config
```

Comment:
I don't quite understand this. If "*-config" utilities come from "depsBuildHost", then `gfortran` should be here too. But then how strictDeps doesn't prevent gfortran from being found, but "*-config" utilities need this?

Reply:
Simple, it seemingly never used gfortran:

> No Fortran extension requested. Continue without fortranext.

I can remove the gfortran bits or try to enable the extension, whichever you prefer.

Reply:
Maybe there was a silent regression at some point. However, I'm almost sure the extension is not useful. The following change worked for me:
```diff
diff --git a/pkgs/by-name/fa/fastnlo-toolkit/package.nix b/pkgs/by-name/fa/fastnlo-toolkit/package.nix
index d43cc2698c1b..6349072ac70f 100644
--- a/pkgs/by-name/fa/fastnlo-toolkit/package.nix
+++ b/pkgs/by-name/fa/fastnlo-toolkit/package.nix
@@ -3,7 +3,6 @@
   stdenv,
   fetchurl,
   boost,
-  gfortran,
   lhapdf,
   ncurses,
   perl,
@@ -38,8 +37,6 @@ stdenv.mkDerivation rec {
   buildInputs =
     [
       boost
-      gfortran
-      gfortran.cc.lib
       lhapdf
       yoda
     ]
```